website/react-magma-docs/src/pages/api/appbar.mdx

```diff
@@ -42,6 +42,50 @@ export function Example() {
 }
 ```
 
+## With Skip Link
+
+The `AppBar` works concurrently with `SkipLink` which assists keyboard navigation.
+
+```tsx
+import React from 'react';
+import {
+  AppBar,
+  AppBarPosition,
+  SkipLink,
+  SkipLinkContent,
+  Paragraph,
+} from 'react-magma-dom';
+
+export function Example() {
+  return (
+    <div style={{ height: '400px', overflow: 'auto', position: 'relative' }}>
+      <AppBar isInverse position={AppBarPosition.sticky}>
+        AppBar content
+      </AppBar>
+      <SkipLink to="#last-one">Skip to the end!</SkipLink>
```

Comment:
Should we add isInverse to the SkipLink so it can be more noticeable? Or remove isInverse from AppBar.

Reply:
Removed it from `SkipLink`